src/hooks/useFlags/flagsmith/flagsmithProvider.tsx

```diff
@@ -0,0 +1,15 @@
+import flagsmith from "flagsmith";
+import { FlagsmithProvider } from "flagsmith/react";
+import React from "react";
+
+export const Provider = ({ children }: { children: React.ReactElement }) => (
+  <FlagsmithProvider
```

Comment:
if I don't have enabled flagsmith, you still initialize it, return just children when there is no flagsmith

Reply:
you can create just one provider eg. FeatureFlagsProvider and switch providers inside of it, more isolation